jenkins-pipelines/perf-regression-throughput-30min-i4i.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys.yaml","configurations/aws/i4i_2xlarge.yaml", "configurations/stress/c-s-throughput-cmds-i4i.yaml"]''',
```

Comment:
the file you created is not being used

Reply:
The job will be created later, if i am correctly understand you

Reply:
now i see that this is a configuration only.